Helper/PaymentDirectHelper.php

```diff
@@ -59,10 +59,10 @@ public function buildTransaction(UserInterface $userDebited, UserInterface $user
         return $transaction;
     }
 
-    public function executeDirectTransaction(UserInterface $userDebited, UserInterface $userCredited, $amount, $fees, $secureModeReturnURL = null)
+    public function executeDirectTransaction(UserInterface $userDebited, UserInterface $userCredited, $amount, $fees, $secureModeReturnURL = 'http://vago.local/app_dev.php/server-time')
```

Comment:
sure, I let it as it was in the original code, so that not to break things silently on your dev environments if any